aurelia-slickgrid/src/aurelia-slickgrid/services/sort.service.ts

```diff
@@ -38,6 +47,7 @@ export class SortService {
       backendApi.preProcess();
     }
     const query = backendApi.service.onSortChanged(event, args);
+    self.emitSortChanged('remote');
```

Comment:
when `self` is removed this can be `this.emitSortChanged('remote')`

Reply:
actually no, the function is a callback (of the Filter DOM element) and has no knowledge of this service anymore, which is why I needed to add another `self` argument